server/test/services/export/QuestionJsonSamplerTest.java

```diff
@@ -34,264 +37,237 @@ public void setUp() {
 
   @Test
```

Comment:
Tests below are functionally the same as before, but we test on the map instead of the string.